src/components/Swap/index.tsx

```diff
@@ -48,21 +48,21 @@ export default function Swap(props: SwapProps) {
 
   return (
     <>
-      <Header title={<Trans>Swap</Trans>}>
```

Comment:
this change is just moving `SwapInfoProvider` higher up, to wrap the settings component too. we need this so that the settings view has access to the current slippage